core/tests/protractor_utils/admin.js

```diff
@@ -78,5 +78,13 @@ var reloadExploration = function(name) {
   });
 };
 
+var reloadAllExplorations = function() {
```

Comment:
Can we delete this? I don't think we want to encourage its use, based on the problems we ran into.